draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+more DNS resolver IP addresses, to be used for traditional unencrypted DNS. However it could also
+be a richer set of information.
 
-During network attach, networks frequently announce a resolver that they recommend clients should use.
-This could be through DHCP, RA, PPP, PCO, or another mechanism. Historically this is usually a Do53 resolver,
-declared simply by its IP address. However it could also be a richer set of information.
+Using an encrypted resolver that is associated to the one provisioned by the network
+can provide several benefits that are not possible if only unencrypted DNS is used:
+
+- Prevent other devices on the network from observing client DNS messages
+- Verify that answers come from the selected DNS resolver
+- Authenticate that the DNS resolver is the one provisioned by the network
+
+Often, network-provisioned resolvers are forwarders running on a local router. The discovered
+encrypted resolvers in these cases may either be local fowarders themselves, or an associated
+resolver that is in the network (thus bypassing the router's DNS forwarder).
 
 | Requirement | Description |
 | R2.1 | Example requirement |
 
 ### Unencrypted forwarder
 
-If the resolver announced by the network is a classic unencrypted forwarder, it is frequently the case that such forwarders are difficult to upgrade
-to support encrypted operation. In such cases it is useful for the resolver provider to be able to declare which encrypted resolvers they provide,
-and for the client to be able to discover them. If the client wishes to, it can then use one of those resolvers and bypass the local forwarder.
+If the resolver announced by the network is a classic unencrypted forwarder, it is frequently the case that such
+forwarders are difficult to upgrade to support encrypted operation. In such cases it is useful for the resolver
+provider to be able to declare which encrypted resolvers they provide, and for the client to be able to discover
+them. If the client wishes to, it can then use one of those resolvers and bypass the local forwarder.
 
 | Requirement | Description |
 | R3.1 | Example requirement |
 
 ### Encrypted forwarder
 
-If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver supports it. Discovering this
-may require communication with the local resolver, or an upstream resolver, over an unencrypted transport. Once discovered, the local encrypted forwarder
-may be selected by the client, gaining the benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local
-topology.
+If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver
+supports it. Discovering this may require communication with the local resolver, or an upstream resolver, over an
+unencrypted transport. Once discovered, the local encrypted forwarder may be selected by the client, gaining the
```

Comment:
You may want to add other benefits like enforcing device specific policies like IoT MUD (RFC8520) and parental control.

Reply:
Agree. Can you contribute text?

Reply:
Sure, you can use the following text for IoT MUD:   

   A common usage pattern for an IoT device is for it to "call home" to
   a service that resides on the public Internet, where that service is
   referenced through a domain name (A or AAAA record).  As discussed in
   Manufacturer Usage Description Specification [RFC8520], because these
   devices tend to require access to very few sites, all other access
   should be considered suspect.  However, if the query is not
   accessible for inspection, it becomes quite difficult for the
   infrastructure to suspect anything.

Reply:
Great; I've applied it.